templates/firefox-ios.xliff

```diff
@@ -3707,6 +3731,279 @@ Private Tab</source>
+      </trans-unit>
+      <trans-unit id="ContextualHints.Shopping.OptedInBody.v120" xml:space="preserve">
+        <source>Are these reviews reliable? Check now to see an adjusted rating.</source>
+        <note>Contextual hints are little popups that appear for the users informing them of new features. This one appears after the user has opted in and informs him if he wants use the review checker by tapping the Shopping button.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.AdjustedRating.Description.v120" xml:space="preserve">
+        <source>Unreliable reviews removed</source>
+        <note>Description adjusted of the rating card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.AdjustedRating.StarsAccessibilityLabel.v120" xml:space="preserve">
+        <source>%@ out of 5 stars</source>
+        <note>Accessibility label, associated to adjusted rating stars. %@ is a decimal value from 0 to 5 that will only use a tenth (example: 3.5).</note>
+      </trans-unit>
+      <trans-unit id="Shopping.AdjustedRating.Title.v120" xml:space="preserve">
+        <source>Adjusted rating</source>
+        <note>Title of the adjusted rating card displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ConfirmationCard.Button.Text.v120" xml:space="preserve">
+        <source>Got It</source>
+        <note>Button text of the confirmation displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.ConfirmationCard.Title.v120" xml:space="preserve">
+        <source>Analysis Is Up To Date</source>
+        <note>Title of the confirmation displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.HighlightsCard.Competitiveness.Title.v120" xml:space="preserve">
+        <source>Competitiveness</source>
+        <note>Section title of the review highlights displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.HighlightsCard.LessButton.Title.v120" xml:space="preserve">
+        <source>Show Less</source>
+        <note>Title of the button that shows less reviews in the review highlights displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.HighlightsCard.MoreButton.Title.v120" xml:space="preserve">
+        <source>Show More</source>
+        <note>Title of the button that shows more reviews in the review highlights displayed in the shopping review quality bottom sheet.</note>
+      </trans-unit>
+      <trans-unit id="Shopping.HighlightsCard.Packaging.Title.v120" xml:space="preserve">
+        <source>Packaging</source>
+        <note>Section title of the review highlights displayed in the shopping review quality bottom sheet.</note>
```

Comment:
I'm not sure what "Packaging" actually refers to here. It seems the comment is too generic, please detail a bit more as to what "packaging" really entails

Reply:
Resolved